core/modules/widgets/image.js

```diff
@@ -52,7 +52,7 @@ ImageWidget.prototype.render = function(parent,nextSibling) {
 		tiddler = this.wiki.getTiddler(this.imageSource);
 	if(!tiddler) {
 		// The source isn't the title of a tiddler, so we'll assume it's a URL
-		src = this.imageSource;
+		src = this.getVariable("tv-get-export-image-link") || this.imageSource;
```

Comment:
I'd expect to be passing `this.imageSource` as a parameter to the `tv-get-export-image-link` macro:

```
src = this.getVariable("tv-get-export-image-link",
  {params: [{name: "title",value: this.imageSource}],
  defaultValue: this.imageSource}
);
```

And similarly for `tv-get-export-link` below.